internal/api/handlers/schedulers_handler_test.go

```diff
@@ -69,7 +68,7 @@ func TestListSchedulers(t *testing.T) {
 		mockCtrl := gomock.NewController(t)
 
 		schedulerStorage := mockports.NewMockSchedulerStorage(mockCtrl)
-		schedulerManager := scheduler_manager.NewSchedulerManager(schedulerStorage, nil, nil, nil)
+		schedulerManager := manager.NewSchedulerManager(schedulerStorage, nil, nil, nil)
```

Comment:
Here what do you think to rename the package `services/scheduler/manager` to be `services/managers`? Inside them you will create the function `NewScheduler` or `NeSchedulerService` and the client will call `managers.NewScheduler` or `managers.NewSchedulerService`